src/app/ui/consoleview.cpp

```diff
@@ -33,10 +34,12 @@ ConsoleView::ConsoleView(QWidget *parent) :
 {
     ui->setupUi(this);
     ui->plainTextEdit->setLocalEchoEnabled(true);
+    ui->plainTextEdit->putData(QByteArray().append("~>"));
+    dirPrinted = true;
     mProcess = new QProcess(this);
     QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
-//    env.insert("PATH", env.value("Path") + ";C:\\Program Files (x86)\\Git\\bin\\;");
-//    env.insert("PATH", env.value("Path") + ";C:\\Program Files (x86)\\Subversion\\bin\\;");
+    //    env.insert("PATH", env.value("Path") + ";C:\\Program Files (x86)\\Git\\bin\\;");
+    //    env.insert("PATH", env.value("Path") + ";C:\\Program Files (x86)\\Subversion\\bin\\;");
```

Comment:
there should be commented code
if u need this  code
you shoud add comment why it is needed